### ISSUE TITLE
Cutting a new release 

### ISSUE THREAD

**Issue by EdwardCooke** [aaubry/YamlDotNet#705]
@aaubry if i was to push a new tag up with the version number would it push a new package out to nuget? I don’t want to put any more changes in until we get what we already have out the door.

**Comment by aaubry** [comment]
There's a different process for that. It's already late here, but I'll explain it tomorrow morning.

**Comment by aaubry** [comment]
I have added information about the release process in the [wiki](https://github.com/aaubry/YamlDotNet/wiki/BuildAndRelease). Please have a look and let me know if it is clear.

**Comment by EdwardCooke** [comment]
Thanks. It looked clear enough. I'm going to follow those instructions in a couple of minutes.

**Comment by EdwardCooke** [comment]
@aaubry It almost worked. Looks like there was a 401 unauthorized talking to nuget.

```text
build: LinkPullRequestsToReleases: System.AggregateException: One or more errors occurred. (Response status code does not indicate success: 401 (Unauthorized).)
 ---> System.Net.Http.HttpRequestException: Response status code does not indicate success: 401 (Unauthorized).
   at System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode()
   at build.BuildDefinition.LinkPullRequestsToReleases(GitVersion version) in C:\projects\yamldotnet\tools\build\BuildDefinition.cs:line 293
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
```

**Comment by aaubry** [comment]
Actually the error happened when calling the GitHub API. The token must have expired. I'll update it for next time.

**Comment by aaubry** [comment]
I've updated the token and executed manually the release step that failed, so that all involved PRs and issues got a message about the release. For the record, the command that I used to do that was:
```
GITHUB_REPOSITORY=aaubry/YamlDotNet GITHUB_TOKEN=******************** ./build.sh --version=12.0.0 LinkPullRequestsToReleases
```
Some notes about the above command:
- `build.sh` is just a script to execute the build. There's also a `build.cmd` to do the same on Windows.
- I had to set the `GITHUB_REPOSITORY` environment variable because by default when the build is executed it uses `aaubry/YamlDotNet.Sandbox`. This is to prevent mistakes and allows to test the build without impacting the real repository.
- The `GITHUB_TOKEN` variable contains a PAT with the `public_repo` permission.
- I forced the version to 12.0.0 because I already had a local commit, so the version was resolving to 12.0.1.
